csharp/src/Ice/OutgoingRequestFrame.cs

```diff
@@ -93,27 +96,30 @@ public sealed class OutgoingRequestFrame : OutgoingFrame
             return request;
         }
 
-        /// <summary>Create a new OutgoingRequestFrame for an operation with multiple parameters or a single struct
-        /// parameter.</summary>
+        /// <summary>Creates a new <see cref="OutgoingRequestFrame"/> for an operation with multiple parameters or a
+        /// single struct parameter.</summary>
+        /// <typeparam name="T">The type of the operation's parameters; it's a tuple type for an operation with multiple
```

Comment:
arguments instead of parameters?

Reply:
I think parameters is correct when talking about an operation: an operation has parameters (and a return type), not arguments and a return value.